config/god/hets_workers.rb

```diff
@@ -6,8 +6,14 @@ def group
   end
 
   def start_cmd
-    hets_opts = ' ' << hets_server_options.join(' ')
-    'exec nice hets -X' << hets_opts
+    hets_opts = ' -X ' << hets_server_options.join(' ')
+    hets_server = Array(HetsSettings['hets_path']).flatten
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.